include/jemalloc/internal/tcache_structs.h

```diff
@@ -49,14 +49,15 @@ struct tcache_slow_s {
 	 */
 	void		*dyn_alloc;
 
+	/* The number of bins activated in the tcache. */
+	unsigned	tcache_nbins;
```

Comment:
The placement for this is not ideal -- it's surrounded by two `void *`s, which means this field has to take 8 bytes even though it only needs half. Place it before `next_gc_bin` would be better.

Reply:
I tried different placement and it seems the size of `tcache_slow_t` always increases from 224 to 232 no matter where I put it. I think this is because the SC_NBINS is 52 and it just combines well with the extra 4 bytes in `next_gc_bin`.  Any suggestions?

Reply:
It's fine if it increases by 8 bytes; the principle is always group same sized fields together (assuming there's no hotness concern). I'd say still put it before next_gc_bin, so if any changes made in the future, chances are they will remain compact.